### ISSUE TITLE
TypeError printing server.schema

### ISSUE THREAD

**Issue by conradoplg** [cannatag/ldap3#169]
I'm following the tutorial and when I print the server schema with
print(server.schema), I get
```
Traceback (most recent call last):
  File "/tmp/server/temp.py", line 5, in <module>
    print(server.schema)
  File "/tmp/.env3/lib/python3.4/site-packages/ldap3/protocol/rfc4512.py", line 167, in __str__
    return self.__repr__()
  File "/tmp/.env3/lib/python3.4/site-packages/ldap3/protocol/rfc4512.py", line 342, in __repr__
    r += ('  Name forms:' + str(self.name_forms))
  File "/tmp/.env3/lib/python3.4/site-packages/ldap3/utils/ciDict.py", line 77, in __str__
    return str(self._store)
  File "/tmp/.env3/lib/python3.4/site-packages/ldap3/protocol/rfc4512.py", line 808, in __repr__
    r = (linesep + '  Object class: ' + self.object_class) if self.object_class else ''
TypeError: Can't convert 'list' object to str implicitly
```

Debugging it I see that self.object_class is a list, which seems to be set as a list in the line
object_def.object_class = oids_string_to_list(value)
but I'm not sure what the correct fix would be (just adding list_to_string?)

**Comment by conradoplg** [comment]
Sorry, submitted by accident, editing it...

**Comment by cannatag** [comment]
You're right. Just add the list_to_string at line 808 of ldap3.protocol.rfc4512.py. I will include the fix in the next release.

Giovanni

**Comment by cannatag** [comment]
will be released in 1.3.0
